### DISCUSSION TITLE
Make Glaze tile Windows on the workspace with the least open windows?

### DISCUSSION THREAD

**Discussion by Wakers** [glzr-io/glazewm#572] [General]
It's late, I don't know if I'm making sense.

Glaze opens every window on workspace 1. Which is fine, to start with, but it would be nice if it could instead open them on workspace 2 when there are 3 or more windows open on workspace 1.

So it isn't just leaving one workspace empty while filling up the other with ever small windows, which for some apps are pointlessly unusable.

Can it do this already?

**Comment by MyGitHubBlueberry** [comment]
It requires `if` conditions to be done. Not sure if yaml can do that.
